client/app/lib/redux/services/payment.coffee

```diff
@@ -36,6 +36,8 @@ exports.fetchSubscription = fetchSubscription = pickData ->
   client.get Endpoints.SubscriptionGet
 
 exports.createSubscription = createSubscription = pickData (params = {}) ->
+  if not params.trialEnd
+    params.trialEnd = (new Date()).getTime() + (30 * 24 * 60 * 60 * 1000)
```

Comment:
minor

we can shorten this with using `Date.now()` which is a shorter version of `(new Date()).getTime()`